cli/cli.go

```diff
@@ -27,13 +27,10 @@ type Interface interface {
 // cli implements cli.Interface
 type cli struct {
 	Configuration
-	version    string
-	build      string
```

Comment:
I don't consider version and build as options. These were read-only through the interface/implementation exposed on cli.Interface and cli.